### ISSUE TITLE
cyc learning rate

### ISSUE THREAD

**Issue by dzwduan** [jefkine/zeta-learn#2]
i think cyc learning rate(https://github.com/titu1994/keras-one-cycle) will be helpful function.
recommend a  cython tutorial   https://github.com/cythonbook/examples.git
